### ISSUE TITLE
Is there support for 2 channel and 4 channel training

### ISSUE THREAD

**Issue by ajithkumarmcw** [Deci-AI/super-gradients#916]
In Yolov8 we can train 2 channel images or 4 channel images by adding `ch:2` or `ch:4` parameter to yolov8s.yaml file . Does Yolo-NAS also have similar support if so where should we change it

**Comment by ofrimasad** [comment]
Hi @ajithkumarmcw .
It seems like we have a bug in not passing the in_channel parameter on instantiation. 
a ticket was opened, and this will be fixed in the coming patch version.

you can work around it by changing the default value in the code of 
`super_gradients.training.models.detection_models.yolo_nas.yolo_nas_variants.YoloNAS_S`
or
`super_gradients.training.models.detection_models.yolo_nas.yolo_nas_variants.YoloNAS_M`
or
`super_gradients.training.models.detection_models.yolo_nas.yolo_nas_variants.YoloNAS_L`

simply change `in_channels: int = 3` to `in_channels: int = 3`

please note that changing the input channels will prevent you from using the pre-trained weights, of course. 
Hope that helps.

**Comment by ajithkumarmcw** [comment]
thanks for adding support
